utilities/virt.py

```
@@ -1550,27 +1550,33 @@ def get_guest_os_info(vmi):
+            for win_os in system_windows_os_matrix
+            for os_name, os_dict in win_os.items()
+            if os_name == windows_version
+        ]
+        if windows_os_dict:
+            return windows_os_dict[0]
+        raise KeyError(f"Failed to extract {windows_version} from system_windows_os_matrix")
+
+    return {}
 
 
 def get_rhel_os_dict(rhel_version):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)